roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -25,9 +25,16 @@
 
 - debug: var=ports
 
+- name: check selinux status
```

Comment:
Got it. Thanks. Anyways we have check for redhat so cleanup task only run for centos. I will remove check selinux status task from ovs_cleanup.yml file